### ISSUE TITLE
Info about pretrained model S3DIS

### ISSUE THREAD

**Issue by youssef962** [mellody11/RandLA-Net-Pytorch-New#25]
Do you have the info about the train/val/test partition rules, point sampling strategy, batch size/epochs, learning rate and class balancing for the your uploaded pretrained model on S3DIS?

**Comment by liuxuexun** [comment]
Sorry for the late reply. I use the default setting in the code.
